### DISCUSSION TITLE
How to plot?

### DISCUSSION THREAD

**Discussion by helderc** [DaveSkender/Stock.Indicators#1072] [Q&A] [answered]
Hi all.

I'm relatively new to C# and I'm wondering which libraries you use to plot these charts.
To me, it seems that Stock.Indicators does a great job in calculating indicators, but does it plot?

If I'm right (Stock.Indicadors doesn't have the plot feature), what do you guys recommend to plot with WPF support?

Thanks in advance.

**Comment by DaveSkender** [comment, marked as answer]
I’ve seen a lot of developers using Plotly lately.
Checkout our existing discussion for [Charting and visualization tools](https://github.com/DaveSkender/Stock.Indicators/discussions/430).

Marking this a duplicate to avoid splintering.

**Reply by helderc** [reply]
Thanks Dave.
